ibu-imager/seedcleanup/seedcleanup.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package seedcleanup
```

Comment:
"Seed Cleanup" kinda sounds like what you would do prior to taking the snapshot, and I personally found it a bit confusing. Perhaps we should call it seed restoration instead?